version7.go

```diff
@@ -38,37 +39,47 @@ func NewV7FromReader(r io.Reader) (UUID, error) {
 		return uuid, err
 	}
 
-	makeV7(uuid[:])
+	makeV7(uuid[:], timeNow())
+	return uuid, nil
+}
+
+// NewV7FromTime returns a Version 7 UUID based on the provided time.
+func NewV7FromTime(t time.Time) (UUID, error) {
```

Comment:
Change this to NewV7WithTime to match NewV6WithTime that is in PR 174

Reply:
Changed